### ISSUE TITLE
firewall-applet user unit 

### ISSUE THREAD

**Issue by denji** [firewalld/firewalld#1415]
### What happened

firewall-applet fails user mode .service unit when not active or disabled.
```
$ systemctl --failed --user
  UNIT                                                                              LOAD   ACTIVE SUB    DESCRIPTION             
● app-\x2fusr\x2fbin\x2ffirewall\x2dapplet@{hash}.service loaded failed failed /usr/bin/firewall-applet
```

### What you expected to happen

Correct termination and operation of systemd firewall-applet states. htop, the "Systemd User" column will be in "degraded" state.

### Firewalld Version

firewalld 2.2.3-1 (pacman)

### Firewalld Backend

FirewallBackend=nftables

### Linux kernel version

6.11.5-zen1-1-zen

**Comment by erig0** [comment]
I have no idea what you're reporting here. `firewall-applet` is not a systemd service.

**Comment by denji** [comment]
@erig0 https://forum.endeavouros.com/t/firewall-applet-constantly-crashes/51358/8
In version 2.1, the service was called slightly differently /etc/xdg/autostart/firewall-applet.desktop (generated). Anyway, rolling back to version 2.0.0 fixed (`sudo downgrade firewalld==2.0.0`) the problem and no longer generates non-working transient unit file

**Comment by denji** [comment]
@erig0 It's an unstable code https://github.com/firewalld/firewalld/pull/1178
